app/views/helpers/TimelineEvents.scala

```diff
@@ -92,12 +93,26 @@ class TimelineEvents @Inject() (
     }
   }
 
-  private def amendmentRejectedAsLatest(submission: Submission): Boolean =
-    submission.latestAction.exists { action =>
-      val isAmendmentRejected = action.requestType == AmendmentRequest
-      val hasLatestErrorNotification = action.notifications.headOption.exists(_.headOption.exists(_.enhancedStatus == ERRORS))
-      submission.blockAmendments == false && isAmendmentRejected && hasLatestErrorNotification
-    }
+  sealed trait AmendmentAsLatest
+  case object AmendmentFailed extends AmendmentAsLatest
+  case object AmendmentRejected extends AmendmentAsLatest
+
+  private def hasAmendmentAsLatest(submission: Submission): Option[AmendmentAsLatest] =
```

Comment:
And this method `hasAmendmentOutcome`

Reply:
But `hasAmendmentOutcome` does not tell if the action/notification is the latest

Reply:
How about `getAmendmentEventIfLatest`?